lib/generate/collectModules.js

```diff
@@ -14,14 +14,11 @@ module.exports = async (page) => {
     /**
      * Use Magento's rjsResolver to wait for all modules to load.
      */
-    await page.evaluate(
-        () =>
-            new Promise((resolve) => {
-                require(['rjsResolver'], (resolver) => {
-                    resolver(() => resolve());
-                });
-            })
-    );
+    await page.evaluate(() => {
+        require(['rjsResolver'], (resolver) => {
+            resolver(() => {});
+        });
+    });
```

Comment:
Can you confirm that this will still wait for require.js to complete? We need to wait until the page has finished loading before we continue. It looks like this change removes the "wait" part of this step.

Reply:
The new code does not maintain the "wait" functionality as the original code. The original code waits for the rjsResolver to signal completion, whereas the new code only waits for the require function to execute, not for all modules to load. However, during my testing, this is where the rjsResolver would hang. I believe you may have ran into a similar issue previously with rjsResolver hanging, according to the comments in blockMagepack.js. I think I can improve upon this. I'll make changes and perform some tests and submit another PR once I've debugged the rjsResolver hanging issue properly. My goal is to not have to block assets due to them causing rjsResolver to hang.